packages/forklift-console-plugin/src/modules/Providers/views/details/components/DetailsSection/components/UIDetailsItem.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useModal } from 'src/modules/Providers/modals';
+import { EditProviderUIModal } from 'src/modules/Providers/modals/EditProviderUI';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { ExternalLink } from '@kubev2v/common';
+import { V1beta1Provider } from '@kubev2v/types';
+
+import { DetailsItem } from '../../../../../utils';
+
+import { ProviderDetailsItemProps } from './ProviderDetailsItem';
+
+/**
+ * @typedef {Object} UIDetailsItemProps - extends ProviderDetailsItemProps
+
+ * @property {string} [text] - A text to be displayed as a content.
+ * @property {function} [calcProviderUI] - A method for auto calculating the provider's UI path.
+ **/
+export interface UIDetailsItemProps extends ProviderDetailsItemProps {
+  text: string;
+  calcProviderUI: (provider: V1beta1Provider) => string;
+}
+
+const getProviderUIAnnotation = (provider: V1beta1Provider): string =>
+  provider?.metadata?.annotations?.['forklift.konveyor.io/providerUI'];
+
+export const UIDetailsItem: React.FC<UIDetailsItemProps> = ({
```

Comment:
it's an 
```ts
<EditableContentButton content={content} onEdit={onEdit} />
```

not an detailsitem

Reply:
I understand your point. Since this component is presented within the providers details section only  so I do want to indicate that it's a details field and not some general field.

All fields within the provider details section are named as the content they include, e.g.:
TypeDetailsItem
NamespaceDetailsItem
URLDetailsItem
CredentialsDetailsItem
OwnerDetailsItem

Part of them are editable, part are not, few display a text and few display a link.
The only uniqueness of this `UIDetailsItem` comp is that it doesn't include a title. 
And  I do agree with your follow up comment regarding the "UI" naming part.

So maybe we can rename it to `WebUIDetailsItemNoTitle` or something like that.

Reply:
I whould like it to be part of the Name DetailItem, the Name details item will have 2 content items, can you make it part of the Name details item content ?